packages/core/src/browser/dialogs.ts

```diff
@@ -23,6 +23,8 @@ import { FrontendApplicationContribution } from './frontend-application';
 @injectable()
 export class DialogProps {
     readonly title: string;
+    maxWidth?: number;
+    wordWrap?: string;
```

Comment:
Perhaps this should be an `enum`?

```ts
/**
* Determine the word wrapping behavior for content in the dialog.
* - `normal`: breaks words at allowed break points.
* - `break-word`: breaks otherwise unbreakable words.
* - `initial`: sets the property to it's default value.
* - `inherit`: inherit this property from it's parent element. 
*/
wordWrap?: 'normal' | 'break-word' | 'initial' | 'inherit';
```